packages/dashboard/src/components/VisualizationsPanel.tsx

```diff
@@ -1,4 +1,3 @@
-import React from 'react'
 import type { Visualization } from '@cdc/core/types/Visualization'
```

Comment:
If time allows: 
There are four type issues in this file that need to be resolved from previous tickets. 

Steps for VS Code:
- Run `lerna run --scope @cdc/dashboard start`
- ctrl/cmd+j to open the terminal
- choose the problems tab
- Filter to Active File Only

Reply:
@adamdoe I can work on this, I just need to know what values should the missing parameters have. 
This was me taking a stab at it. If there is somewhere I can look to get the default values then I will take care of it. 
```
newVisualizationConfig.general = {
boxplot: null, 
type: "", 
geoType: "", 
showDownloadButton: true, 
showMissingDataLabel: false, 
showSuppressedSymbol: false, 
showZeroValueDataLabel: false
}
```
And same with tableConfig it is missing the following values
```
fontSize: ,
showMissingDataLabel: , 
showSuppressed: 

```

As for the other 2 issues, I need to add them to the Type